src/openrct2-ui/windows/Ride.cpp

```diff
@@ -2919,54 +2919,36 @@ static void window_ride_vehicle_paint(rct_window *w, rct_drawpixelinfo *dpi)
 
     // Capacity
     gfx_draw_string_left(dpi, STR_CAPACITY, &rideEntry->capacity, COLOUR_BLACK, x, y);
-    if (font_get_line_height(FONT_SPRITE_BASE_MEDIUM) == 10) {
-        // Excitement Factor
-        factor = rideEntry->excitement_multiplier;
-        if (factor > 0)
-        {
-            y += 10;
-            gfx_draw_string_left(dpi, STR_EXCITEMENT_FACTOR, &factor, COLOUR_BLACK, x, y);
-        }
-
-        // Intensity Factor
-        factor = rideEntry->intensity_multiplier;
-        if (factor > 0)
-        {
-            y += 10;
-            gfx_draw_string_left(dpi, STR_INTENSITY_FACTOR, &factor, COLOUR_BLACK, x, y);
-        }
 
-        // Nausea Factor
-        factor = rideEntry->nausea_multiplier;
-        if (factor > 0)
-        {
-            y += 10;
-            gfx_draw_string_left(dpi, STR_NAUSEA_FACTOR, &factor, COLOUR_BLACK, x, y);
-        }
+    // Excitement Factor
+    factor = rideEntry->excitement_multiplier;
+    if (factor > 0)
+    {
+        y += LIST_ROW_HEIGHT;
```

Comment:
@AaronVanGeffen  thanks for amending. but could you check for italian? italian has no extra margin in ui range. so if you add 12 to y, it will be out of ui range. my code add 10 to all y values in case font_get_line_height(FONT_SPRITE_BASE_MEDIUM) == 10
please check for this

Reply:
Hmm, I had tested with several languages, just not Italian. You're right, though — it is using four lines instead of two or three. Well, we'll have to tackle that at a later point — I had just merged this PR when you were typing your comment.

I suspect a further change won't be necessary, though, as the string is probably going to change soon. (This is already apparent when you compare the English version its the American English counterpart, for example.)